### ISSUE TITLE
App is running with backup file only, no app logs possible

### ISSUE THREAD

**Issue by jung-eto** [DeFiCh/app#561]
#### What happened:
**Case 1:**
Updated to v2.3.0
Synchronisation successfully done
used latest Backup to restore wallet from /users/myname/Downloads/backup.dat (suggestion from the app)
app restored successfully, all fund are there
closed app
deleted backup file from downloads folder
opened app again
app is in "Create or restore wallet" mode again (see screenshot)

**Case 2:**
downloaded backup again
restored wallet successfully again
wallet stays open
deleted backup file from downloads folder
tried to unlock wallet
got error message (see screenshot)
if I insert my backup file in the directory again with running app and try to unlock it then, everything works fine

#### What you expected to happen:
after restoring the wallet with my backup file I want to delete the backup file from my computer as I do not want to leave the backup file on my computer for security reasons. I am used to have the app running and starting from the wallet.dat file in ~/Library/Application Support/DeFi/wallets and not from a backup file.
And I also want to be able to unlock my wallet without having a corresponding backup file somewhere on my computer

#### How to reproduce it (as minimally and precisely as possible):
1.: Updated to v2.3.0
2.: Synchronisation successfully done until 100%
3.: used latest Backup to restore wallet from /users/myname/Downloads/backup.dat
4.: close the app (wait until it is shut down correctly)
5.: delete backup file from directory used in step 3
6.: open the app again
7.: app is in "Create or restore wallet" mode again (see screenshot)

#### What are your environment parameters?

- DeFi App Version: v2.3.0
- Operating System (OS): macOS 11.2.1 (MacBook Pro, M1 Chip)

#### Anything else we need to know?:

Unfortunately it is impossible for me to save any log files since the update to v2.3.0 
I tried to do so with fully restored wallet in locked and unlocked state.
The only thing that happened was that a folder "database" was created in my directory from step 3 with a file in it named "log.0000000001"
Here are screenshots of all cases I described above:

**Case 1: opened the app again after restoring it successfully, with no backup file in directory**
<img width="1013" alt="Bildschirmfoto 2021-02-25 um 12 47 32" src="https://user-images.githubusercontent.com/78017139/109206319-fc3c8300-77a7-11eb-920b-24f34728a131.png">

**Case 2: Tried to unlock wallet after restoring it successfully, with no backup file in directory**
<img width="656" alt="Bildschirmfoto 2021-02-25 um 12 38 38" src="https://user-images.githubusercontent.com/78017139/109206389-170ef780-77a8-11eb-8e7b-33ee6a3861ad.png">

**Comment by thedoublejay** [comment]
Hi @jung-eto, from your current scenario, you're trying to load the backup in some folder. Actually, the app is using that file to load your funds. If you move/delete this file, the node won't be able to find this file resulting into rolling back into an unloaded state (Create or Restore screen).

I suggest is you load your wallet from a secured folder instead of downloads. Then, make sure that you have it encrypted. When your wallet is encrypted, no transactions will work (e.g, sending, DEX, add liquidity etc.).

**Comment by jung-eto** [comment]
Hi @thedoublejay , thank you for your fast reply! Ok, I did not know that this behaviour is meant to be as it is. I followed the directory the app suggested to. How can I manage to have a restored wallet running without a backup file? My backup files are encrypted. 
And how can I manage to be able to create log files for you again?
Thank you :)

**Comment by thedoublejay** [comment]
Hi @jung-eto, no worries. Can you explain a bit what you're trying to do? Sorry it's not very clear to me. When you restore a backup file, it's the wallet itself.

For the other issue related to logs, what issues are you facing? When you click on Logs -> App logs and select a folder. Does it open the file selection screen? Does it generate a log file?

**Comment by jung-eto** [comment]
Hi @thedoublejay , 
"When you restore a backup file, it's the wallet itself." --> I did not know that. In older app versions I could restore my wallet with the backup file, then remove the backup file from my computer and the wallet worked fine from then on, because the app used its wallet.dat in Library/...
Now I tried to do so again. It seems as if you changed something in the backup file handling.

"For the other issue related to logs, what issues are you facing? When you click on Logs -> App logs and select a folder. Does it open the file selection screen? **--> Yes, the screen opens**
Does it generate a log file? **--> No, no log file at all. Nothing happens.**
